scripts/mkimage

```diff
@@ -267,6 +267,15 @@ elif [ "$BOOTLOADER" = "u-boot" ]; then
   cleanup
 fi # bootloader
 
+if [ "$UBOOT_VERSION" != "hardkernel" ]; then
```

Comment:
This condition will be true for RPi, Generic etc. - basically anything that doesn't have `UBOOT_VERSION="hardkernel"` will execute this, and fail.

Did you mean to include this block in one of the "u-boot" related conditions above, or at least process this block only if a suitable UBOOT option variable is set?

@Kwiboos suggestion might cleaner.